eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/JavaDocFormatting.java

```diff
@@ -0,0 +1,89 @@
+// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+package com.azure.tools.checkstyle.checks;
+
+import com.puppycrawl.tools.checkstyle.api.DetailNode;
+import com.puppycrawl.tools.checkstyle.api.JavadocTokenTypes;
+import com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck;
+import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
+
+/**
+ *  Description text should only have one space character after the parameter name or '@return' statement.
+ *  Text should not start on a new line or have any additional spacing or indentation.
+ */
+public class JavaDocFormatting extends AbstractJavadocCheck {
+
+    private static final String JAVA_DOC_RETURN = "javadoc return";
+    private static final String JAVA_DOC_PARAMETER = "javadoc parameter";
+    private static final String JAVA_DOC_THROW = "javadoc throw";
+
+    private static final String ERROR_THROW_WRONG_FORMAT = "Wrong format for javadoc throw statement";
```

Comment:
What is the right format?

Reply:
This is for an unrecognized format on a Javadoc throws..  something like 
`@throws` or `@throws 1 something`.

But, on second though, Javadoc parser would throw exception in that case, so, I will remove this better